grpc.go

```diff
@@ -665,6 +665,8 @@ func (a *agentGRPC) TtyWinResize(ctx context.Context, req *pb.TtyWinResizeReques
 }
 
 func (a *agentGRPC) CreateSandbox(ctx context.Context, req *pb.CreateSandboxRequest) (*gpb.Empty, error) {
+	agentLog.WithField("grpc-request", req).Debug()
```

Comment:
This should not be part of this PR, right ??

Reply:
Oops! Updated ;)